client-v2/overrides.d.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  CommonFieldProps,
+  WrappedFieldProps,
+  Formatter,
+  Normalizer,
+  Parser,
+  Validator,
+  GenericFieldHTMLAttributes
+} from 'redux-form';
+import { ComponentType, Component } from 'react';
+
+declare module 'redux-form' {
+  interface BaseFieldProps<P = {}> extends Partial<CommonFieldProps> {
+    name: string;
+    // Any-typed to avoid a known issue, see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/26253
```

Comment:
Sadly 😢 Any suggestions welcome - I didn't spend too much time on this but maybe with a pair we could do better here